modules/eap-xp-10-env/1.0.0/module.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "eap-xp-10-env"
+description: "JBoss Enterprise Application Platform XP 1.0 env and labels"
+version: "1.0.0"
+
+labels:
+    - name: "org.jboss.product"
+      value: "eap-xp"
+    - name: "org.jboss.product.version"
+      value: "1.0"
+    - name: "org.jboss.product.eap.version"
+      value: "7.3.1"
+    - name: "com.redhat.deployments-dir"
+      value: "/opt/eap/standalone/deployments"
+    - name: "com.redhat.dev-mode"
+      value: "DEBUG:true"
+      description: "Environment variable used to enable development mode (debugging). A value of true will enable development mode."
+    - name: "com.redhat.dev-mode.port"
+      value: "DEBUG_PORT:8787"
+      description: "Environment variable used to specify the debug port. If not set, the default EAP debug port will be used (8787). Only applicable when development mode is enabled."
+envs:
+    - name: "WILDFLY_VERSION"
+      value: "7.3.0.GA-redhat-SNAPSHOT"
+    - name: "LAUNCH_JBOSS_IN_BACKGROUND"
+      value: "true"
+    - name: "JBOSS_PRODUCT"
+      value: "eap-xp"
+    - name: "JBOSS_EAP_VERSION"
+      value: "7.3.1"
+    - name: "PRODUCT_VERSION"
+      value: "1.0"
```

Comment:
same remark as above (`1.0.0`)

Reply:
Resolved in https://github.com/jboss-container-images/jboss-eap-7-image/pull/185